rclcpp/include/rclcpp/node.hpp

```diff
@@ -1039,6 +1039,9 @@ class Node : public std::enable_shared_from_this<Node>
    * rcl_interfaces::msg::SetParametersResult to indicate whether or not the
    * parameter should be set or not, and if not why.
    *
+   * The back returns a shared pointer to the callback handle.
+   * The callback is valid as long as the smart pointer is alive.
```

Comment:
```suggestion
   * Users should retain a copy of the returned shared pointer, otherwise the callback will be removed.
   * I.e. the callback is valid as long as the smart pointer is alive.
```